content/ops/managing-users.md

```diff
@@ -5,6 +5,12 @@ menu:
 title: Managing Users
 ---
 
+## Note for GovCloud environment
+
+Only single sign-on user accounts are allowed. Service accounts, such as deployer credentials, are permissible with the condition that they are scoped to a particular space with limited access.
```

Comment:
We definitely should have language like that if this is not an ops page, but I think it is...